core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/Run.java

```diff
@@ -48,241 +98,550 @@
  * @author Rob Davey
  * @since 0.0.2
  */
-@JsonSerialize(typing = JsonSerialize.Typing.STATIC)
-@JsonInclude(JsonInclude.Include.NON_NULL)
-@JsonTypeInfo(use = JsonTypeInfo.Id.CLASS, include = JsonTypeInfo.As.PROPERTY, property = "@class")
-@JsonIgnoreProperties({ "securityProfile", "submissionDocument" })
-public interface Run
```

Comment:
It's not redundant. Coding to interfaces is standard practice and very useful when you need to swap implementations. It's also convenient for multiple inheritance. e.g. DetailedSampleBuilder takes advantage of this with the Sample interfaces, and a similar RunBuilder could simplify some things down the road